RELEASING.md

```diff
@@ -280,7 +272,13 @@ See the documentation on the [JanusGraph release policy](https://docs.janusgraph
 
 *This is the point of no return.* After releasing artifacts to Maven
 Central and pushing history to the public GitHub repo, the release
-can't be canceled.
+can't be canceled. 
+
+Update the `STRUCTOR_LATEST_TAG` variable on Travis-CI and 
+restart build on Travis to ensure that 
```

Comment:
_restart_ ? Shouldn't a new build be started?

I know this is really nitpicking, but the releasing docs should be a clear step by step instruction any release manager can simply follow.